components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -528,13 +528,13 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
           </div>
         </div>
       </div>
-      <Dialog
-        onOpenChange={() => {
-          setIsDivFocused(false);
-        }}
```

Comment:
We don't need this. It actually is problematic in general. If you have a modal open, it will close automatically if you click accidentally outside the modal.